index.html

```diff
@@ -753,10 +752,10 @@ <h2 id="3Definitions">Definitions</h2>
 <dt>luminance</dt></dfn>
 
 <dd>formal definition of luminance is in [[COLORIMETRY]].
-Informally it is the perceived brightness, or <a href=
-"#3greyscale"><span class="Definition">greyscale</span></a>
-level, of a colour. Luminance and <a href="#3chromaticity"><span
-class="Definition">chromaticity</span></a> together fully define
+Informally it is the perceived brightness, or 
+<span class="Definition">[=greyscale=]</span>
+level, of a colour. Luminance and <span
+class="Definition">[=chromaticity (CIE)=]</span> together fully define
```

Comment:
This changes the wording ever so slightly. It links to the same destination as the old spec. But now the wording includes that "(CIE)".

There is perhaps a way to avoid the wording change if we really want. But it might be fine. Let me know your thoughts.

Reply:
I think this is a beneficial change